index.html

```diff
@@ -11,7 +11,76 @@
+                            <li><a href="./about.html">English</a></li>
+                            <li><a href="./about.html">My Page</a></li>
+                            <li><a href="./about.html">Logout</a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="container-nav">
+                    <div class="logo-nav">
+                        <a href="./index.html">
+                            <img src="assets/images/logo.png" alt="logo of VG Global">
+                        </a>
+                    </div>
+                    <ul class="header-nav-desktop">
+                        <li><a href="./about.html">About</a></li>
+                        <li><a href="./about.html">Program</a></li>
+                        <li><a href="./about.html">Join</a></li>
+                        <li><a href="./about.html">Sponsor</a></li>
+                        <li><a href="./about.html">News</a></li>
+                        <li><a href="./about.html">CC Campaign</a></li>
+                    </ul>
+                </div>
+            </nav>
+            <div class="header-nav-phone">
+                <button type="button" class="menu-btn">
+                    <img src="assets/images/hamburger-icon.svg" alt="menu link" id="hamburger">
+                </button>
+                <nav class="header-nav-mobile hidden">
+                    <button type="button" class="btn-close-nav">
+                        <img src="assets/images/icon-cancel.svg" alt="cancel icon">
+                    </button>
+                    <ul>
+                        <li>
+                            <a href="./about.html" class="link-about">
+                                About
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="link-program">
+                                Program
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="link-join">
+                                Join
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="link-sponsor">
+                                Sponsor
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="link-news">
+                                News
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="link-cc-campaign">
+                                CC Campaign
+                            </a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>            
             <div class="headline">
```

Comment:
- Kindly your headline is not part of the `header`, please make it out of the `header` and inside its own `section` to make more semantically correct 👍👍